### DISCUSSION TITLE
Update notification UI in vscode to help highlight new integrations like WSL2

### DISCUSSION THREAD

**Discussion by satyapramodh** [microsoft/Windows-Dev-Performance#34] [General]
# Environment
Windows 10, 2004 
v.19041.450

VS Code:
Version: 1.48.0 (user setup)
Commit: db40434f562994116e5b21c24015a2e40b2504e6
Date: 2020-08-13T07:50:42.600Z
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.19041

# Description
After installing windows 10 2004 , I enabled WSL2 but I was not aware of how vs code extensions remote-wsl, remote-containers existed/worked. @bitcrazed [pointed to me](https://twitter.com/richturn_ms/status/1293602764484825089?s=20) on twitter regarding the VS notification toast that may have triggered.

I realized then that, for my daily uses I ignore/dismiss most (useful or noise) vs code toasts as i feel i get a lot of them and it became a habit. And so i missed the WSL related notification. With [Rich's idea](https://twitter.com/richturn_ms/status/1293651712893280256?s=20), I got to thinking how that experience can be improved. Dynamic notification UI based on the notification type might help general user's brains figure out noise from important ones. 

My general idea of notifications types are:
1. File extension is undetected by file format renderer and so suggestion to look for extensions on the marketplace
2. Extension crash errors
3. Each extension's notifications? (i think this is a thing?)
4. More amazing notifications I cant remember

# Proposed Behavior
Here are possible, silly notification/toast UI

![image](https://user-images.githubusercontent.com/3414042/90441500-e712c900-e08d-11ea-8ec9-18bbe4f2fece.png)
vscode notification showing WSL-ubuntu related info
![image](https://user-images.githubusercontent.com/3414042/90441512-ed08aa00-e08d-11ea-815d-a735343718af.png)
vscode notification showing python lang related info

1. Have colored headers/borders to indicate important vs non important or by notification category.
2. Images to tell if its vscode notification or an extension(each ext icon).. dev can then decide what to register in their heads as important. Example: I care about WSL-ubuntu more than WSL open suse and their separate icons is all i need to pay attention.
2. We could also have a project level dashboard which has all the important notifications, suggestions, errors listed, segregable for devs to check on anytime as opposed to oops i dismissed that notification, now that info is lost forever.

# Actual behavior
![image](https://user-images.githubusercontent.com/3414042/90441071-30164d80-e08d-11ea-9ebf-6276050e6de5.png) 
current vscode extension ui

**Comment by bitcrazed** [comment]
Thanks for filing. May move/re-create on the VSCode repo since this is VSCode specific, but happy to have this issue land here first, either way.
